ui/src/router/components/configuration/components/EnricherConfigSection.js

```diff
@@ -3,7 +3,7 @@ import { EuiPanel } from "@elastic/eui";
 import { DockerConfigViewGroup } from "./docker_config_section/DockerConfigViewGroup";
 
 export const EnricherConfigSection = ({ config: { enricher } }) => {
-  return !enricher ? (
+  return !enricher || enricher.type === "nop" ? (
```

Comment:
Out of curiosity, is this something that should have been done previously or?

Reply:
Good question - it's needed with the new changes for constructing the Turing router object from the API response, before it is used in the details view. We have [this](https://github.com/gojek/turing/blob/main/ui/src/services/router/TuringRouter.js#L29) to capture the default values for the Docker config form, in the enricher definition. So the enricher config is not empty and we need to explicitly check its type now.